manifests/init.pp

```diff
@@ -219,6 +219,22 @@
 #
 # @param udev_purge_rules
 #   Toggle if unmanaged files in /etc/udev/rules.d should be purged if manage_udevd is enabled
+#
+# @param manage_system_conf
+#   Should system service manager configurations be managed
+#
+# @param system_settings
+#   Config Hash that is used to configure settings in system.conf
+#   NOTE: It's currently impossible to have multiple entries of the same key in
```

Comment:
https://github.com/puppetlabs/puppetlabs-inifile/issues/41

Reply:
Yep, I found it.. still not implemented 🤷🏻‍♂️